src/components/console/RequestList/Item.js

```diff
@@ -0,0 +1,40 @@
+import { useStyles } from './Item.style'
+import { useConsoleContext } from 'src/context/ConsoleContext'
+import React from 'react'
+import clsx from 'clsx'
+import { useScrollWithDropdown } from './useScrollWithDropdown'
+import dots from '../../../icons/dots.svg'
+import green from '../../../icons/EllipseGreen.svg'
+import red from '../../../icons/EllipseRed.svg'
+
+function Item({name, success, customref, onDelete}) {
+    const {deleteRequest, copyRequest, doRequest} = useConsoleContext()
+    const styles = useStyles()
+    const [buttonRef, ulRef] = useScrollWithDropdown();
+
+    return (
+        <div  className={styles.rel} >
+            <div className={styles.imgWrap}>
+                {success ? <img src={green} alt="green" />: <img src={red} alt="red" />}
```

Comment:
Format the ternary operator in the right way please! As you did here https://github.com/Sa6aTeur/API-console/pull/1/files#diff-05c96ccfe038b092ee29604adbbfa12d42f32fe5441fe760e3bc39227c60b3f4R51

Reply:
this comment is done

Reply:
LGTM